tests/unit/amo/components/TestAddon.js

```diff
@@ -1238,6 +1239,46 @@ describe(__filename, () => {
 
     expect(root.find('title')).toHaveLength(0);
   });
+
+  describe('Contribute', () => {
+    it('does not render a Contribute card when there is no add-on', () => {
+      const root = shallowRender({ addon: null, params: { slug: 'some-slug' } });
+
+      expect(root.find(Contribute)).toHaveLength(0);
+    });
+
+    it('does not render a Contribute card when there is no contributions URL', () => {
+      const contributionsURL = null;
+
+      const { store } = dispatchClientMetadata();
+      const addon = createInternalAddon({
+        ...fakeAddon,
+        contributions_url: contributionsURL,
+      });
+      store.dispatch(_loadAddons({ addon }));
+
+      const root = renderComponent({ params: { slug: addon.slug }, store });
+
+      expect(root.find(Contribute)).toHaveLength(0);
+    });
+
+    it('renders a Contribute card when add-on has a contributions URL', () => {
+      const contributionsURL = 'https://paypal.me/babar';
+
+      const { store } = dispatchClientMetadata();
+      const addon = createInternalAddon({
+        ...fakeAddon,
+        contributions_url: contributionsURL,
+      });
+      store.dispatch(_loadAddons({ addon }));
+
+      const root = renderComponent({ params: { slug: addon.slug }, store });
+
+      expect(root.find(Contribute)).toHaveLength(1);
+      expect(root.find(Contribute))
+        .toHaveProp('contributionsURL', contributionsURL);
```

Comment:
This test should validate all other properties too: `addonType`, `numberOfAuthors`.